__tests__/src/services/upstream/response.js

```diff
@@ -0,0 +1,38 @@
+import { api_proto as apiProto } from '../../../../src/services/upstream/compiled';
+
+const { Response } = apiProto;
+
+const message = {
+
+  dRKeyStoreGet: {
+    messageKey: 1,
```

Comment:
As I already said, those values are invalid. Please checkout the protbuf and test with real values.